macros/udfs/sentiment_analysis.sql

```diff
@@ -0,0 +1,36 @@
+{% macro create_udf_sentiment_analysis() %}
+
+create or replace function {{target.schema}}.udf_sentiment_analysis(text STRING, output INTEGER)
+returns STRING
```

Comment:
your arguments need revision
text STRING, model INTEGER or STRING [more on this later], output VARIANT (if you still want to differentiate between label and score)

and returns STRING, you have to make sure L32 has to cast as a string.